### ISSUE TITLE
Theme Selector : <select> options are not styleable on Firefox v49.0.1+

### ISSUE THREAD

**Issue by SethClydesdale** [SethClydesdale/forumactif-edge#12]
**Topic on forum :** http://fmdesign.forumotion.com/t1055-12-pick-a-theme#22340

**Description :** A handful of people have reported inconsistencies in the style of the option elements for the theme selector. Normally the theme options display the color of the theme in the background of the option, but after update 49.0.1 on Firefox, some people have reported the options being all white.

**Todo :** Since this has been an issue for awhile, I've decided that it's time to change the theme selector to something that is NOT a <select> element. Preferably a DIV that contains anchors as the options. It'll no doubt require an overhaul of the script though -- replacement via update might be tricky too. I'll update this with concepts when I get to work on 'em.

**Concerned File(s) :**
- [all.js:L70](https://github.com/SethClydesdale/forumactif-edge/blob/master/javascripts/in-all-the-pages/all.js#L70)

**Comment by SethClydesdale** [comment]
We've let our Firefox users deal with this issue long enough. It's about time we've patched this issue, so those afflicted with this bug can view the theme color that they're about to choose. I've already written up a draft which will replace that current `<select>` element with an anchor and a div. This will update the theme selector UI on ALL browsers, so that we don't have anymore inconsistencies cross-browser-wise.

Overall the design wont change too much, and the result is almost completely identical to a standard `<select>` element. The perks of this change is that Administrators will be able to restyle the theme selector **completely** with custom designs, transitions, etc... I'll leave an image below which shows the pseudo-select element ( on the left ) and standard select element ( on the right ) with the style rendering bug on Firefox.
![](https://i58.servimg.com/u/f58/18/21/60/73/captur10.png)

I'm going to be test running the draft on our development / support forum @ [FM Design](http://fmdesign.forumotion.com/). Once I've confirmed everything is working as it should I'll prep this plugin for the next patch.
